docs/guide/installation/20_zigbee2mqtt-fails-to-start_crashes-runtime.md

```diff
@@ -208,3 +208,34 @@ This happens when you edit one or more of the `pan_id`, `network_key` or `ext_pa
 ```
 ext_pan_id: [0x39,0xaf,0x4d,0x83,0xh2,0xdc,0xb3,0x89]
 ```
+
+## Error: runtime crash with `status=HASH_ERROR_TIMEOUTS` or fail to start with `status=HOST_FATAL_ERROR`
```

Comment:
The error is `ASH_ERROR_TIMEOUTS` not `HASH_ERROR_TIMEOUTS`.
`HOST_FATAL_ERROR` is also too generic to put in this context (it can have many, many causes).

I think it's best to remove the references to ember-specific errors since I'm sure the same applies to other stacks. Maybe a header like this: `Regular crashes with timeout or failure to start after serial opened`.

Reply:
My bad for the typo.
I agree that a generic header will help more than references to ember-specific errors.
I will change the header as per your suggestion.